src/Hosting/TestHost/test/Utilities.cs

```diff
@@ -3,27 +3,16 @@
 
 using System;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Testing;
 
 namespace Microsoft.AspNetCore.TestHost
 {
     internal static class Utilities
```

Comment:
We should delete these in another PR

Reply:
https://github.com/aspnet/AspNetCore/issues/16620